packages/core/lib/engine_http.js

```diff
@@ -167,7 +167,7 @@ HttpEngine.prototype.createScenario = function (scenarioSpec, ee) {
     });
   });
 
-  return self.compile(tasks, scenarioSpec.flow, ee);
+  return self.compile(tasks, scenarioSpec, ee);
```

Comment:
`compile` was being passed `scenarioSpec.flow` but not doing anything with it, therefore the change here doesn't really change anything, and we can now use the whole object.